phpunit.xml.dist

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit colors="true" bootstrap="vendor/autoload.php">
+    <server name="SYMFONY_PHPUNIT_VERSION" value="7.5" />
```

Comment:
I'd rather use 8.4, that's latest phpunit

Reply:
WFM :+1: